src/pods/toolbar/components/export-button/export-button.business.ts

```diff
@@ -43,3 +43,46 @@ export const expandAllFieldsInTables = (table: TableVm[]) =>
     ...table,
     fields: expandTableFields(table.fields),
   }));
+
+// Export Schema functions
+const getPropertyJsonSchema = (field: FieldVm): string => {
+  if (field.isArray) {
+    return `"${field.name}": { bsonType: "array", items: { bsonType: "${field.type}" } }`;
+  }
+
+  if (field.children && field.children.length > 0) {
+    const properties = getPropertiesJsonSchema(field.children, false);
+    return `"${field.name}": { bsonType: "object", title: "${field.name}", properties: { ${properties}, }, }`;
+  }
+  return `"${field.name}": { bsonType: "${field.type}" }`;
+};
+
+const getPropertiesJsonSchema = (fields: FieldVm[], useTab = true): string => {
+  const separator = useTab ? ',\n\t\t\t\t' : ', ';
+  return fields.map(getPropertyJsonSchema).join(separator);
+};
+
+// TODO -> Implement this function, when NN feature is implemented
+// const getRequiredFields = (fields: FieldVm[]): string[] => {}
```

Comment:
NN implementation is almost there,once is implemented maybe you can guide an student into implementing this